lib/exec.js

```diff
@@ -39,31 +39,25 @@ const PATH = require('./utils/path.js')
 // runScript({ pkg, event: 'npx', ... })
 // process.env.npm_lifecycle_event = 'npx'
 
-class Exec {
-  constructor (npm) {
-    this.npm = npm
+class Exec extends BaseCommand {
+  /* istanbul ignore next - see test/lib/load-all-commands.js */
+  static get name () {
+    return 'exec'
   }
 
-  get usage () {
-    return usageUtil('exec',
-      'Run a command from a local or remote npm package.\n\n' +
-
-      'npm exec -- <pkg>[@<version>] [args...]\n' +
-      'npm exec --package=<pkg>[@<version>] -- <cmd> [args...]\n' +
-      'npm exec -c \'<cmd> [args...]\'\n' +
-      'npm exec --package=foo -c \'<cmd> [args...]\'\n' +
-      '\n' +
-      'npx <pkg>[@<specifier>] [args...]\n' +
-      'npx -p <pkg>[@<specifier>] <cmd> [args...]\n' +
-      'npx -c \'<cmd> [args...]\'\n' +
-      'npx -p <pkg>[@<specifier>] -c \'<cmd> [args...]\'' +
-      '\n' +
-      'Run without --call or positional args to open interactive subshell\n',
-
-      '\n--package=<pkg> (may be specified multiple times)\n' +
-      '-p is a shorthand for --package only when using npx executable\n' +
-      '-c <cmd> --call=<cmd> (may not be mixed with positional arguments)'
-    )
+  /* istanbul ignore next - see test/lib/load-all-commands.js */
+  static get description () {
+    return 'Run a command from a local or remote npm package.'
+  }
+
+  /* istanbul ignore next - see test/lib/load-all-commands.js */
+  static get usage () {
+    return [
+      '-- <pkg>[@<version>] [args...]',
+      '--package=<pkg>[@<version>] -- <cmd> [args...]',
+      '-c \'<cmd> [args...]\'',
+      '--package=foo -c \'<cmd> [args...]\'',
+    ]
```

Comment:
This is usage, not the man page.  This just shows the args you can use.  Because it was free-form before it got a little muddied.  If I ask for usage on command a, I don't really need to see how to use a completely different command.  If we want a "see also" we can talk about that but I really think that belongs in `help`

Reply:
Having each flag describe what it does is definitely the direction this is moving us towards. For now we are separating our man page from basic usage. Next iteration we can define args instead of calling this "usage"

Usage is something that is built up from components i.e. command name, aliases, command description, args and their descriptions, etc.